src/stream/src/executor/monitor/streaming_stats.rs

```diff
@@ -183,6 +183,23 @@ pub struct StreamingMetrics {
     pub kv_log_store_buffer_unconsumed_epoch_count: LabelGuardedIntGaugeVec<4>,
     pub kv_log_store_buffer_unconsumed_min_epoch: LabelGuardedIntGaugeVec<4>,
 
+    pub sync_kv_log_store_total_read_count: LabelGuardedIntCounterVec<4>,
```

Comment:
Seems too verbose to have a new field for each detailed metrics. We may try merging several related metrics and later distinguish them in grafana by their label value. For example, to merge `xxx_total_read_count` and `xxx_buffer_read_count`, we can just have one `xxx_read_count` and add a more label named like `type` on it, and when we create the labeled value, we can add the `type` (`total`, `buffer`) to the label list.

At least `xxx_read_size`, `xxx_read_count`, `xxx_write_size`, `xxx_write_count`, `(clean/unclean)_state` can be merged.